build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
> So we run tests differently

If there's a real difference between `dnx test` and `dnvm run default test` then a `dnvm` issue has regressed recently.

> the original exit code is **lost**

@BrennanConroy fixed aspnet/dnvm#428 a while ago. I haven't seen any evidence of a regression there. But  reopen that issue if it's a problem again.